tracer/src/Datadog.Trace/AppSec/Rasp/MetaStructHelper.cs

```diff
@@ -0,0 +1,95 @@
+// <copyright file="MetaStructHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Vendors.MessagePack.Formatters;
+
+#nullable enable
+
+namespace Datadog.Trace.AppSec.Rasp;
+
+internal static class MetaStructHelper
+{
+    public static Dictionary<string, object> StackTraceInfoToDictionary(string? type, string language, string id, string? message, List<Dictionary<string, object>> frames)
+    {
+        var dict = new Dictionary<string, object>(3)
+        {
+            { "language", language },
+            { "id", id },
+            { "frames", frames }
+        };
+
+        if (type is not null && type.Length > 0)
```

Comment:
You can simplify this to: 
```suggestion
        if (type is { Length: > 0 })
```

But people differ on whether they prefer it, so up to you 😄

Reply:
I will go for it, I find it more simple. Thanks!